examples/scripts/cycling_ageing_yang.py

```diff
@@ -0,0 +1,36 @@
+import pybamm as pb
+
+options = {"sei": "ec reaction limited",
+           "sei film resistance": "distributed",
+           "porosity": "variable porosity"}
```

Comment:
Note that the "sei film resistance" is set automatically by the following lines in `BaseBatteryModel`:
```python3
        # Change the default for SEI film resistance based on which sei option is
        # provided
        extra_options = extra_options or {}
        sei_option = extra_options.get("sei", None)  # return None if option not given
        if sei_option is None:
            default_options["sei film resistance"] = None
        else:
            default_options["sei film resistance"] = "distributed"
```
Something similar could be done for porosity

Reply:
Thanks for the tip: I will erase the redundant setting for sei film resistance in this example. In the case of porosity, It would be a nice feature, but the variable porosity is not a default option for SEI models (older publications like Ramadass et. al. 2003 don't include it), so I think that is better to leave it as a "visible" option.